README.md

```diff
@@ -30,6 +30,7 @@ You can also do `python -m cve_bin_tool.cli <flags> <path to directory>` which i
 
   Functional options:
   -x, --extract         autoextract compressed files
+  -v, --verbose         details on found issues as script runs
```

Comment:
As with the MANUAL.md changes, the -v option is gone and should be removed here.